src/Avalonia.Controls/Control.cs

```diff
@@ -84,6 +84,13 @@ public class Control : InputElement, IControl, INamed, IVisualBrushInitialize, I
                 nameof(Unloaded),
                 RoutingStrategies.Direct);
 
+        /// <summary>
+        /// Defines the <see cref="SizeChanged"/> event.
+        /// </summary>
+        public static readonly RoutedEvent<SizeChangedEventArgs> SizeChangedEvent =
+            RoutedEvent.Register<Control, SizeChangedEventArgs>(
+                nameof(SizeChanged), RoutingStrategies.Bubble);
```

Comment:
I'd use `RoutingStrategies.Direct` which also matches WPF/WinUI. IMO this event doesn't need to propagate up or down the tree.

Reply:
Good catch! I didn't even think to check that. It should help performance too, at least theoretically.

Reply:
Fixed